### DISCUSSION TITLE
Feature Request: Custom opengraph images for private and organisation profiles

### DISCUSSION THREAD

**Discussion by real-yfprojects** [community/community#128966] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

Github allows for each repository to set a custom *social media preview* image corresponding to the `og:image` tag in the opengraph protocol. This is great!

However I would like to have the same feature for user and organisation profiles.

Currently the open graph image will be set the users profile picture which has a square aspect ratio and therefore hardly fits the usual size in social media previews. Therefore I would like to create a custom open graph image that better fits the needs of social media preview.
The same applies for organization profiles.

I would really appreciate such an enhancement. Thanks a lot in advance!

**Comment by ebndev** [comment]
Hi @real-yfprojects, thanks for being a part of the GitHub Community!

We sincerely appreciate you taking the time to share your thoughts.

As a valued GitHub user, your feedback in this Discussion space, along with appropriate tags and examples, plays a crucial role in shaping the development of GitHub. While we can’t guarantee immediate action on every piece of feedback, please know that we are attentive to your suggestions and continuously strive to enhance the GitHub experience.
